presto-main/src/main/java/io/prestosql/FullStatementRewriteContext.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
please update commit message

Reply:
All commits squashed with commit message "pluggable query modification support for data access control"